fix lint error, removing unused codes in JS

soniawu opened the pull request:
I did lint clean up on HTML and CSS. On JS, I removed for debug only console.log(), removed the if (data != null) { } else {}, because those are not working. Since lint doesn't let us to use cap letter, I had to change variables like zipCity to zip-city across all files. Please take a look.

soniawu committed:
fix lint error, removing unused codes in JS

mjachna commented:
So is the line towards the end handing all the error that may happen since you removed the if else that was supposed to handle errors? Otherwise looks great.

soniawu commented:
The (data != null) was put in early. I thought that was the way to catch error, but it didn't work. Later I found out jquery do success and fail callback separate. Look at what I added later,  .done() and .fail(). Null data will never get into the .done(). So I cleaned them up.  If you would like to capture the sunrise sunset API json error, you can do the same thing.

Sent from my iPhone

> On Nov 30, 2016, at 11:27 PM, mjachna <notifications@github.com> wrote:
> 
> So is the line towards the end handing all the error that may happen since you removed the if else that was supposed to handle errors? Otherwise looks great.
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub, or mute the thread.
>

mjachna commented:
**# Oh okay thanks for explaining that to me.**
